react/bootstrap/src/components/select/select.tsx

```diff
@@ -51,32 +40,23 @@ function Badges<Item>({slotContext}: {slotContext: SelectContext<Item>}) {
 	return badges.length ? <>{badges}</> : null;
 }
 
+function SelectItem<Item>({itemContext, slotContext}: {itemContext: ItemContext<Item>; slotContext: SelectContext<Item>}) {
+	const classname = ['dropdown-item position-relative'];
```

Comment:
We already include the library [classnames](https://www.npmjs.com/package/classnames) that allows to make this kind of code cleaner. Like shown:
```suggestion
	const classname = classNames('dropdown-item', 'position-relative', {'text-bg-primary': slotContext.state.highlighted});
```